torchvision/models/mobilenet.py

```diff
@@ -138,12 +145,15 @@ def __init__(self, num_classes=1000, width_mult=1.0, inverted_residual_setting=N
                 nn.init.normal_(m.weight, 0, 0.01)
                 nn.init.zeros_(m.bias)
 
-    def forward(self, x):
+    def _forward(self, x):
```

Comment:
Oh, interesting way of doing subclassing.

Another possible approach would be to do something like
```python
_forward = forward
```
like [we do in the DataLoader](https://github.com/pytorch/pytorch/blob/e3e54282cd83624d3b1004c8a580751c75e07710/torch/utils/data/dataloader.py#L351), although I'm not sure if TorchScript supports this? If not, this is fine

Reply:
Can you do it the other way around, so the original implementation uses `def forward` and we add a line creating a duplicated method named `_forward`?